app/assets/v2/js/pages/dashboard.js

```diff
@@ -14,6 +14,8 @@ var sidebar_keys = [
   'misc'
 ];
 
+document.results_limit = 50;
```

Comment:
Does this need to be stored at the document level as opposed to 
`dashboard.results_limit` ?

Reply:
updated